### DISCUSSION TITLE
Virtual monitors, remove on disconnection

### DISCUSSION THREAD

**Discussion by thib3113** [rustdesk/rustdesk#8044] [General]
Hello,

I'm using 1.2.4 with virtual monitors, (and it works really well, separates windows are amazing too, thank you) . 

But, I think it miss an option to automatically remove them when disconnected, so, on remote I can use multiples monitors, and physically I don't have a ghost monitors . 

Thank you

**Comment by rustdesk** [comment]
@fufesou let's confirm how the other softwares do this before taking action.

- [x] If remove by default
- [x] How to design the option? local option or display option on controlling side or an option on controlled side

**Comment by rustdesk** [comment]
https://github.com/rustdesk/rustdesk/pull/9023

**Comment by rustdesk** [comment]
Please test the nightly build.

**Reply by thib3113** [reply]
yes, it works, thank you :)
